src/assets/styles/_type.scss

```diff
@@ -1,18 +1,16 @@
 // Type
-$line-height: 1.3rem !default;
-$base-font-size: 14px !default;
-
 :root {
-  --calcite-app-line-height: #{$line-height};
+  --calcite-app-line-height: 1.125rem;
 
-  --calcite-app-base-font-size: #{$base-font-size};
+  --calcite-app-base-font-size: 16px;
```

Comment:
I'd recommend setting the root font size to 10px, that way the rem multiplication math is really simple.

Reply:
Metric system font sizing :P

Reply:
That'd be conceptually clean.
Browsers often default to 16px as their base font, so we can ride on that.
Also, this is based on a loooooooong discussion around typography with the calcite-components and Calcite peeps.

Reply:
As a note, the typography scaling for the majority of our type will be `0.125rem`.

Reply:
👍